catalog/ui/search-ui/standard/src/main/webapp/js/view/MetacardDetail.view.js

```diff
@@ -153,6 +167,12 @@ define([
             }
         });
 
+        Metacard.NearbyLocationRegion = new Marionette.Region({
```

Comment:
I don't think you need to define this region. It creates a region for you automatically when you add a region to "regions" in the layout view.